app/src/main/java/com/weatherxm/data/repository/AppConfigRepository.kt

```diff
@@ -13,6 +13,8 @@ interface AppConfigRepository {
     fun setLastRemindedVersion()
     fun getInstallationId(): String?
     fun getRewardsHideAnnotationThreshold(): Long
+    fun isMainnetEnabled(): Boolean
```

Comment:
Also, we could use a Pair<Boolean, String> fun result here, for the featureEnabled and the message variable.

Reply:
I agree, but we may need `isMainnetEnabled` for other changes as well (e.g. the text changes we had discussed when we get on mainnet) so that's why I created a specific, short-lived function.